hist/hist/src/THLimitsFinder.cxx

```diff
@@ -381,8 +383,8 @@ void THLimitsFinder::OptimizeLimits(Int_t nbins, Int_t &newbins, Double_t &xmin,
       xmin = -1;
       xmax = 1;
    } else {
-      xmin    = binlow;
-      xmax    = binhigh;
+      xmin = std::min(binlow, xmin);
```

Comment:
```suggestion
      xmin = std::min(binlow, xmin - 0.01 * (xmax - xmin));
```
Or leaving as is, but then changing below 0.01 to 0.001 so that the asymmetry is no longer visible by eye